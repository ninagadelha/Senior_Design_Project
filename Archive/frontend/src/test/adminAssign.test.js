import renderer from "react-test-renderer";
import React, { useState } from "react";

import AdminAssign from "../routes/adminAssign";

jest.mock("../styles/home.css");
jest.mock("../index.css");
jest.mock("../components/footer");
jest.mock("../components/AdminNavBar");
jest.mock("react-router-dom");
jest.mock("axios");

const renderTree = (tree) => renderer.create(tree);
describe("<AdminAssign>", () => {
  it("should render component", () => {
    expect(renderTree(<AdminAssign />).toJSON()).toMatchSnapshot();
  });
});
