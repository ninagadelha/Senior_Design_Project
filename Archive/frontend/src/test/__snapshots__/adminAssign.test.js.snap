// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AdminAssign> should render component 1`] = `
<div
  class="cont"
>
  <div
    class="content-placeholder-results"
  >
    <h1>
      Assign Admin
    </h1>
    <div>
      <label>
        Enter Account Email: 
        <input
          id="emailInput"
          onChange={[Function]}
          type="email"
        />
      </label>
    </div>
    <div>
      <button
        className="action-button"
        onClick={[Function]}
        style={
          {
            "marginTop": "16px",
          }
        }
      >
        Assign Admin
      </button>
    </div>
  </div>
</div>
`;
